.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.0/containers/javascript-node
+{
+  "name": "NextJS Dev Container",
```

Comment:
Not sure if dots are allowed but this is the right way to write `Next.js` <picture data-single-emoji=":thumbs:" title=":thumbs:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..CXLFQTYYwn38q20D.ONpZv8AUN3CZccqIF1OaCebHkKsNOaaENYEdXkN0IpSRotAal1ZsgXFPcfgLr61tG3HEG5J0QIPQ8tz41jpbfJDoVzZviXd5Qr8RLWUn-LaJfGjaS876kg.DKCTnqrgzo2N3Z8GLttMYw" alt=":thumbs:" align="absmiddle"></picture> 
```suggestion
  "name": "Next.js Dev Container",
```